.github/workflows/test.yml

```diff
@@ -33,6 +33,7 @@ jobs:
         exclude:
```

Comment:
Could you exclude 1.9 on 24.04? (see linked issue)
I scheduled a second run on ruby-builder but I guess it will fail too.